src/Core/hcoio_messy_mod.F90

```diff
@@ -379,6 +379,23 @@ SUBROUTINE HCO_MESSY_REGRID ( HcoState,  NcArr,               &
     ! Get horizontal grid directly from HEMCO state
     lon   => HcoState%Grid%XEDGE%Val(:,1)
     lat   => HcoState%Grid%YEDGE%Val(1,:)
+
+    ! Optional debug print: print what is used to create dest axis for location on grid
```

Comment:
@lizziel: This needs to be commented out (it was probably uncommented for testing).

Reply:
Thanks for catching that! Now fixed.